irohad/ametsuchi/impl/postgres_block_index.hpp

```diff
@@ -31,32 +31,21 @@ namespace iroha {
      public:
       explicit PostgresBlockIndex(soci::session &sql);
 
-      void index(const shared_model::interface::Block &block) override;
-
-     private:
-      /**
-       * Make index account_id -> list of blocks where his txs exist
-       * @param account_id of transaction creator
-       * @param height of block
-       */
-      auto indexAccountIdHeight(const std::string &account_id,
-                                const std::string &height);
-
       /**
-       * Collect all assets belonging to creator, sender, and receiver
-       * to make account_id:height:asset_id -> list of tx indexes (where
-       * tx with certain asset is placed in the block)
-       * @param account_id of transaction creator
-       * @param height of block
-       * @param index of transaction in the block
-       * @param commands in the transaction
+       * Create several indices for block. Namely:
+       * transaction hash -> block, where this transaction is stored
+       * transaction creator -> block where his transaction is located
```

Comment:
It may be useful to mention "account_id:height -> list of tx indexes" index here as well.